spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/pom.xml

```diff
@@ -1,20 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<artifactId>spring-cloud-data-module-deployer-cloudfoundry</artifactId>
-	<packaging>jar</packaging>
-	<name>spring-cloud-data-module-deployer-cloudfoundry</name>
-	<description>module deployer SPI implementation for Cloud Foundry</description>
-	<parent>
-		<groupId>org.springframework.cloud</groupId>
-		<artifactId>spring-cloud-data-module-deployers-parent</artifactId>
-		<version>1.0.0.BUILD-SNAPSHOT</version>
-	</parent>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-data-module-deployer-spi</artifactId>
-		</dependency>
-	</dependencies>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-cloud-data-module-deployer-cloudfoundry</artifactId>
```

Comment:
Just a thought: how about having the artifactId (and this project name) as `spring-cloud-data-module-deployer-cf` which could help shortening the name a bit ;)